cmd/oras/blob/push.go

```diff
@@ -42,7 +42,7 @@ type pushBlobOptions struct {
 func pushCmd() *cobra.Command {
 	var opts pushBlobOptions
 	cmd := &cobra.Command{
-		Use:   "push [flags] name[@digest] file",
+		Use:   "push [flags] <name>[@digest] <file>",
```

Comment:
I just found a UX inconsistency here.

We should remove the `--media-type` option. Instead, we should
```suggestion
		Use:   "push [flags] <name>[@digest] <file>[:<type>]",
```

/cc @qweeah @lizMSFT @yuehaoliang-microsoft @FeynmanZhou